docs-src/app-dev-context/custom-payload-conversion.md

```diff
@@ -0,0 +1,19 @@
+---
+id: custom-payload-conversion
+title: How to use custom payload conversion
+description: Create your custom `PayloadConverter` and set it on a `DataConverter` in your Client options.
+sidebar_label: Custom payload conversion
+tags:
+  - guide-context
+---
+
+Most SDKs provide a [`PayloadConverter`](/concepts/what-is-a-payload-converter) that can be customized to convert values in custom data types to bytes and back.
+
+Creating a custom Payload Converter is optional.
+It's needed only if the default Data Converter does not support your custom values.
+
+Create your [custom `PayloadConverter`](/concepts/what-is-a-payload-converter#custom-payload-conversion) and set it on a `DataConverter` in your Client options.
```

Comment:
Doesn't sound right. At least, it's not cross-SDK correct text. 
In Java PayloadConverter and DataConverter are two completely different types, you can't set PayloadConverter as a DataConverter. You need to configure a DataConverter (DefaultDataConverter) using this PayloadConverter.

Reply:
Updated to 
`Create your [custom `PayloadConverter`](/concepts/what-is-a-payload-converter#custom-payload-conversion) and configure the `DataConverter` to use it in your Client options.`